packages/@react-aria/combobox/docs/useComboBox.mdx

```diff
@@ -315,433 +323,184 @@ function Option({item, state, shouldUseVirtualFocus}) {
+    {id: 5, email: 'yourEmail@email.com'},
+    {id: 6, email: 'valid@email.com'},
+    {id: 7, email: 'spam@email.com'},
+    {id: 8, email: 'newsletter@email.com'},
+    {id: 9, email: 'subscribe@email.com'}
+  ];
 
-  if (isSelected) {
-    backgroundColor = 'blueviolet';
-    color = 'white';
-  } else if (isFocused) {
-    backgroundColor = 'gray';
-    color = 'black';
-  } else if (isDisabled) {
-    backgroundColor = 'transparent';
-    color = 'gray';
-  }
+  let {startsWith} = useFilter({sensitivity: 'base'});
+  let [filterValue, setFilterValue] = React.useState('');
+  let filteredItems = React.useMemo(
+    () => options.filter((item) => startsWith(item.email, filterValue)),
+    [options, filterValue]
+  );
 
   return (
-    <li
-      {...optionProps}
-      ref={ref}
-      style={{
-        background: backgroundColor,
-        color: color,
-        padding: '2px 5px',
-        outline: 'none',
-        cursor: 'pointer'
-      }}>
-      {item.rendered}
-    </li>
+    <ComboBox
+      label="To:"
+      items={filteredItems}
+      inputValue={filterValue}
+      onInputChange={setFilterValue}
+      placeholder="Enter recipient email"
+      allowsCustomValue>
+      {(item) => <Item>{item.email}</Item>}
+    </ComboBox>
   );
 }
-
-///- end collapse -///
-// Using the same ComboBox component code from the previous example...
-
-<ComboBox label="Favorite Animal" defaultInputValue="red">
-  <Item key="red panda">Red Panda</Item>
-  <Item key="cat">Cat</Item>
-  <Item key="dog">Dog</Item>
-  <Item key="aardvark">Aardvark</Item>
-  <Item key="kangaroo">Kangaroo</Item>
-  <Item key="snake">Snake</Item>
-</ComboBox>
 ```
 
-### Controlled
+### Fully controlled
 
 The following example shows how you would create a controlled ComboBox, controlling everything from the selected value (`selectedKey`)
-to the combobox options (`items`). By passing in `inputValue`, `selectedKey`, `isOpen`, and `items` to the `useComboBoxState` hook you can control
+to the combobox options (`items`). By passing in `inputValue`, `selectedKey`, `isOpen`, and `items` to the `ComboBox` you can control
 exactly what your ComboBox should display. For example, note that the item filtering for the controlled ComboBox below now follows a "starts with"
 filter strategy, accomplished by controlling the exact set of items available to the ComboBox whenever the input value updates.
 
 It is important to note that you don't have to control every single aspect of a ComboBox. If you decide to only control a single property of the
 ComboBox, be sure to provide the change handler for that prop as well e.g. controlling `selectedKey` would require `onSelectionChange` to be passed to `useComboBox` as well.
 
 ```tsx example
-///- begin collapse -///
-function ComboBox(props) {
-  // Get a basic "contains" filter function for input value and option text comparison
-  let {contains} = useFilter({sensitivity: 'base'});
+function ControlledComboBox() {
```

Comment:
The ControlledComboBox doesn't render, digging to see why
![image](https://user-images.githubusercontent.com/9661127/105897523-22b2a700-5fcd-11eb-9208-8f08e6648c19.png)